components/create_comment/create_comment.jsx

```diff
@@ -122,7 +122,17 @@ export default class CreateComment extends React.PureComponent {
         /**
          * Set if channel is read only
          */
-        readOnlyChannel: PropTypes.bool
+        readOnlyChannel: PropTypes.bool,
+
+        /**
+         * Set if @channel should warn in this channel.
+         */
+        enableConfirmNotificationsToChannel: PropTypes.bool.isRequired,
+
+        /**
```

Comment:
This is the type of comment that spawned our conversation on pre-release. For now, these PRs effectively copy the local style of the files being modified, in the hopes that we can go back and holistically rip these out.